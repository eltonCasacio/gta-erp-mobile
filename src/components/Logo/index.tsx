import React from 'react';
import * as S from './styles';

const Logo = () => {
  return (
    <S.Wrapper
      source={require('../../assets/logo.png')}
      accessibilityLabel="Logo da GTA"
    />
  );
};

export default Logo;
