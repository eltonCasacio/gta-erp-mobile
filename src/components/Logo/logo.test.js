import React from 'react';
import renderer from 'react-test-renderer';

import Logo from '.';

describe('<Logo />', () => {
  it('should render default component', () => {
    const tree = renderer.create(<Logo />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
