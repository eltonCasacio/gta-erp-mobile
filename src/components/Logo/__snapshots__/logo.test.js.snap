// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render default component 1`] = `
<Image
  accessibilityLabel="Logo da GTA"
  source={
    Object {
      "testUri": "../../../src/assets/logo.png",
    }
  }
  style={
    Array [
      Object {},
    ]
  }
/>
`;
